nunit.templates/source.extension.vsixmanifest

```diff
@@ -1,35 +1,35 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-    <Metadata>
-        <Identity Id="nunit.templates.b29e95b8-bf26-4f03-983d-f1d0f21ad6ef" Version="1.5" Language="en-US" Publisher="Rob Prouse" />
```

Comment:
If you edit the vsixmanifest using the Visual Studio designer, it will go back to four spaces. Can we put an editorconfig rule in to indent .vsixmanifest at 4 spaces so that it doesn't go back and forth>

(https://github.com/nunit/nunit-vs-templates/pull/44#discussion_r240012449)

Reply:
It has this rule:
[*]
indent_style = space
indent_size = 4

which should do it.

Do you mean 2 spaces ?

Reply:
Got it, it is included with the other 4 spacers....  will fix